text="""PT P
PN WO2018107129-A1; US2018340218-A1; US2018340219-A1; BR112018015871-A2; US10266887-B2; US10266886-B2; AU2017371324-A1; WO2018107129-A8; CA3049961-A1; KR2019104030-A; US2019309357-A1; EP3551753-A1; CN110506128-A; JP2020501546-W
TI Nucleic acid detection system for detecting target nucleic acids and polypeptide in samples, comprises a detection clustered regularly interspaced short palindromic repeats (CRISPR) system comprising an effector protein and guide RNAs.
AU ABUDAYYEH O
   COLLINS J J
   GOOTENBERG J
   ZHANG F
   LANDER E S
   COLLINS J
AE BROAD INST INC (BROD-C)
   MASSACHUSETTS INST TECHNOLOGY (MASI-C)
   HARVARD COLLEGE (HARD-C)
   BROAD INST INC (BROD-C)
   MASSACHUSETTS INST TECHNOLOGY (MASI-C)
   HARVARD COLLEGE (HARD-C)
   BROAD INST INC (BROD-C)
   MASSACHUSETTS INST TECHNOLOGY (MASI-C)
   HARVARD COLLEGE (HARD-C)
   BROAD INST INC (BROD-C)
   MASSACHUSETTS INST TECHNOLOGY (MASI-C)
   HARVARD COLLEGE (HARD-C)
GA 201846551F
AB    NOVELTY - Nucleic acid detection system comprises a detection clustered regularly interspaced short palindromic repeats (CRISPR) system comprising an effector protein and guide RNAs designed to bind to corresponding target molecules; and RNA-based masking construct.
   USE - The nucleic acid detection system is useful for detecting target nucleic acids and polypeptide in samples (all claimed); and useful in multiple scenarios in human health including, for example, viral detection, bacterial strain typing, sensitive genotyping, and detection of disease-associated cell free DNA.
   ADVANTAGE - The RNA targeting effectors provide a robust CRISPR-based diagnostic with attomolar sensitivity. The system can detect both DNA and RNA with comparable levels of sensitivity and can differentiate targets from non-targets based on single base pair differences. Moreover, the system can be prepared in freeze-dried format for convenient distribution and point-of-care (POC) applications.
   DETAILED DESCRIPTION - INDEPENDENT CLAIMS are included for:
   (1) a polypeptide detection system comprising a detection CRISPR system comprising an effector protein and guide RNAs designed to bind to a trigger RNA; RNA-based masking construct; and detection aptamers comprising a masked RNA polymerase promoter binding site or a masked primer binding site;
   (2) a diagnostic device comprising individual discrete volumes, each individual discrete volume comprising the CRISPR system;
   (3) detecting target nucleic acids in samples, which involves distributing a sample or set of samples into individual discrete volumes, the individual discrete volumes comprising the CRISPR system; incubating the sample or set of samples under conditions sufficient to allow binding of the guide RNAs to target molecules; activating the CRISPR effector protein via binding of the guide RNAs to the target molecules, where activating the CRISPR effector protein results in modification of the RNA-based masking construct such that a detectable positive signal is generated; and detecting the detectable positive signal, where detection of the detectable positive signal indicates a presence of target molecules in the sample; or detecting a target nucleic acid in a sample, which involves contacting a sample with the nucleic acid detection system; and applying the contacted sample to a lateral flow immunochromatographic assay; and
   (4) detecting polypeptides in samples, which involves distributing a sample or set of samples into a set of individual discrete volumes, the individual discrete volumes comprising peptide detection aptamers, and CRISPR system; incubating the sample or set of samples under conditions sufficient to allow binding of the peptide detection aptamers to the target molecules, where binding of the aptamer to a corresponding target molecule exposes the RNA polymerase binding site or primer binding site resulting in generation of a trigger RNA; activating the RNA effector protein via binding of the guide RNAs to the trigger RNA, where activating the RNA effector protein results in modification of the RNA-based masking construct such that a detectable positive signal is produced; and detecting the detectable positive signal, where detection of the detectable positive signal indicates a presence of target molecules in a sample.
DC A89 (Photographic, laboratory equipment, optical - including electrophotographic, thermographic uses.); B04 (Natural products and polymers. Including testing of body fluids (other than blood typing or cell counting), pharmaceuticals or veterinary compounds of unknown structure, testing of microorganisms for pathogenicity, testing of chemicals for mutagenicity or human toxicity and fermentative production of DNA or RNA. General compositions.); C06 (Biotechnology - including plant genetics and veterinary vaccines.); D16 (Fermentation industry - including fermentation equipment, brewing, yeast production, production of pharmaceuticals and other chemicals by fermentation, microbiology, production of vaccines and antibodies, cell and tissue culture and genetic engineering.)
MC A12-V03C2; A12-W11L; B04-B03C; B04-C01; B04-E01; B04-E13; B04-E99; B04-L05A2; B05-A03B; B06-A03; B06-F05; B11-C01E; B11-C07B3; B11-C08D2; B12-K04F; B12-K04G; B14-D07; B14-L06; B14-S18; C04-B03C; C04-C01; C04-E01; C04-E13; C04-E99; C04-L05A2; C05-A03B; C06-A03; C06-F05; C11-C01E; C11-C07B3; C11-C08D2; C12-K04F; C12-K04G; C14-D07; C14-L06; C14-S18; D05-H09; D05-H18B; D05-H19C; D05-H99
IP C12N-015/09; C12N-015/10; C12N-015/11; C12Q-001/68; C12N-015/113; C12N-009/22; C12P-019/34; C12Q-001/6806; C12Q-001/6816; C12Q-001/6869; C12Q-001/6876; C12Q-001/6809; C12Q-001/6811; G01N-033/00; C12Q-001/6886; C12Q-001/689; C12Q-001/6895; G01N-033/53; G01N-033/569; G01N-033/68; C12M-001/34; C12N-015/115; C12Q-001/04; C12Q-001/6844
PD WO2018107129-A1   14 Jun 2018   C12N-015/09   201844   Pages: 312   English
   US2018340218-A1   29 Nov 2018   C12Q-001/6869   201880      English
   US2018340219-A1   29 Nov 2018   C12Q-001/6869   201880      English
   BR112018015871-A2   22 Jan 2019   C12N-015/09   201911      English
   US10266887-B2   23 Apr 2019   C12Q-001/6809   201930      English
   US10266886-B2   23 Apr 2019   C12Q-001/6809   201930      English
   AU2017371324-A1   11 Jul 2019   C12N-015/09   201952      English
   WO2018107129-A8   11 Jul 2019   C12N-015/09   201953      English
   CA3049961-A1   14 Jun 2018   C12N-015/09   201958      English
   KR2019104030-A   05 Sep 2019   C12Q-001/68   201970      
   US2019309357-A1   10 Oct 2019   C12Q-001/6869   201978      English
   EP3551753-A1   16 Oct 2019   C12N-015/09   201980      English
   CN110506128-A   26 Nov 2019   C12Q-001/6886   201995      Chinese
   JP2020501546-W   23 Jan 2020   C12N-015/09   202008   Pages: 202   Japanese
AD WO2018107129-A1    WOUS065477    08 Dec 2017
   US2018340218-A1    US917546    09 Mar 2018
   US2018340219-A1    US917549    09 Mar 2018
   BR112018015871-A2    BR11015871    08 Dec 2017
   US10266887-B2    US917549    09 Mar 2018
   US10266886-B2    US917546    09 Mar 2018
   AU2017371324-A1    AU371324    08 Dec 2017
   WO2018107129-A8    WOUS065477    08 Dec 2017
   CA3049961-A1    CA3049961    08 Dec 2017
   KR2019104030-A    KR719231    08 Dec 2017
   US2019309357-A1    US285128    25 Feb 2019
   EP3551753-A1    EP879489    08 Dec 2017
   CN110506128-A    CN80086203    08 Dec 2017
   JP2020501546-W    JP531062    08 Dec 2017
FD  US2018340218-A1 Provisional Application US432553P
   US2018340218-A1 Provisional Application US456645P
   US2018340218-A1 Provisional Application US471930P
   US2018340218-A1 Provisional Application US484869P
   US2018340218-A1 Provisional Application US568268P
   US2018340218-A1 Cont of Application WOUS065477
   US2018340219-A1 Provisional Application US432553P
   US2018340219-A1 Provisional Application US456645P
   US2018340219-A1 Provisional Application US471930P
   US2018340219-A1 Provisional Application US484869P
   US2018340219-A1 Provisional Application US568268P
   US2018340219-A1 Cont of Application WOUS065477
   BR112018015871-A2 PCT application Application WOUS065477
   BR112018015871-A2 Based on Patent WO2018107129
   US10266887-B2 Provisional Application US432553P
   US10266887-B2 Provisional Application US456645P
   US10266887-B2 Provisional Application US471930P
   US10266887-B2 Provisional Application US484869P
   US10266887-B2 Provisional Application US568268P
   US10266887-B2 Cont of Application WOUS065477
   US10266887-B2 Previous Publ. Patent US2018340219
   US10266886-B2 Provisional Application US432553P
   US10266886-B2 Provisional Application US456645P
   US10266886-B2 Provisional Application US471930P
   US10266886-B2 Provisional Application US484869P
   US10266886-B2 Provisional Application US568268P
   US10266886-B2 Cont of Application WOUS065477
   US10266886-B2 Previous Publ. Patent US2018340218
   AU2017371324-A1 PCT application Application WOUS065477
   AU2017371324-A1 Based on Patent WO2018107129
   CA3049961-A1 PCT application Application WOUS065477
   CA3049961-A1 Based on Patent WO2018107129
   KR2019104030-A PCT application Application WOUS065477
   KR2019104030-A Based on Patent WO2018107129
   US2019309357-A1 Provisional Application US432553P
   US2019309357-A1 Provisional Application US456645P
   US2019309357-A1 Provisional Application US471930P
   US2019309357-A1 Provisional Application US484869P
   US2019309357-A1 Provisional Application US568268P
   US2019309357-A1 Cont of Application US917546
   US2019309357-A1 Cont of Application WOUS065477
   US2019309357-A1 Cont of Patent US10266886
   EP3551753-A1 PCT application Application WOUS065477
   EP3551753-A1 Based on Patent WO2018107129
   CN110506128-A PCT application Application WOUS065477
   CN110506128-A Based on Patent WO2018107129
   JP2020501546-W PCT application Application WOUS065477
   JP2020501546-W Based on Patent WO2018107129
PI US432553P    09 Dec 2016
   US456645P    08 Feb 2017
   US471930P    15 Mar 2017
   US484869P    12 Apr 2017
   US568268P    04 Oct 2017
   US917546    09 Mar 2018
   US917549    09 Mar 2018
   BR11015871    02 Aug 2018
   US285128    25 Feb 2019
   KR719231    02 Jul 2019
   CA3049961    11 Jul 2019
   CN80086203    09 Aug 2019
DS WO2018107129-A1: 
		      (National): AE; AG; AL; AM; AO; AT; AU; AZ; BA; BB; BG; BH; BN; BR; BW; BY; BZ; CA; CH; CL; CN; CO; CR; CU; CZ; DE; DJ; DK; DM; DO; DZ; EC; EE; EG; ES; FI; GB; GD; GE; GH; GM; GT; HN; HR; HU; ID; IL; IN; IR; IS; JO; JP; KE; KG; KH; KN; KP; KR; KW; KZ; LA; LC; LK; LR; LS; LU; LY; MA; MD; ME; MG; MK; MN; MW; MX; MY; MZ; NA; NG; NI; NO; NZ; OM; PA; PE; PG; PH; PL; PT; QA; RO; RS; RU; RW; SA; SC; SD; SE; SG; SK; SL; SM; ST; SV; SY; TH; TJ; TM; TN; TR; TT; TZ; UA; UG; US; UZ; VC; VN; ZA; ZM; ZW
      (Regional): BW; GH; GM; KE; LR; LS; MW; MZ; NA; RW; SD; SL; ST; SZ; TZ; UG; ZM; ZW; EA; AL; AT; BE; BG; CH; CY; CZ; DE; DK; EE; ES; FI; FR; GB; GR; HR; HU; IE; IS; IT; LT; LU; LV; MC; MK; MT; NL; NO; PL; PT; RO; RS; SE; SI; SK; SM; TR; OA
WO2018107129-A8: 
		      (National): AE; AG; AL; AM; AO; AT; AU; AZ; BA; BB; BG; BH; BN; BR; BW; BY; BZ; CA; CH; CL; CN; CO; CR; CU; CZ; DE; DJ; DK; DM; DO; DZ; EC; EE; EG; ES; FI; GB; GD; GE; GH; GM; GT; HN; HR; HU; ID; IL; IN; IR; IS; JO; JP; KE; KG; KH; KN; KP; KR; KW; KZ; LA; LC; LK; LR; LS; LU; LY; MA; MD; ME; MG; MK; MN; MW; MX; MY; MZ; NA; NG; NI; NO; NZ; OM; PA; PE; PG; PH; PL; PT; QA; RO; RS; RU; RW; SA; SC; SD; SE; SG; SK; SL; SM; ST; SV; SY; TH; TJ; TM; TN; TR; TT; TZ; UA; UG; US; UZ; VC; VN; ZA; ZM; ZW
      (Regional): AL; AT; BE; BG; BW; CH; CY; CZ; DE; DK; EA; EE; ES; FI; FR; GB; GH; GM; GR; HR; HU; IE; IS; IT; KE; LR; LS; LT; LU; LV; MC; MK; MT; MW; MZ; NA; NL; NO; OA; PL; PT; RO; RS; RW; SD; SE; SI; SK; SL; SM; ST; SZ; TR; TZ; UG; ZM; ZW
EP3551753-A1: 
		      (Regional): AL; AT; BE; BG; CH; CY; CZ; DE; DK; EE; ES; FI; FR; GB; GR; HR; HU; IE; IS; IT; LI; LT; LU; LV; MC; MK; MT; NL; NO; PL; PT; RO; RS; SE; SI; SK; SM; TR; BA; ME; MA; MD; TN
CP WO2018107129-A1
      US20160282354-A1      
      WO2015148431-A1   IMMCO DIAGNOSTICS INC (IMMC-Non-standard)   SURESH L, MALYAVANTHAM K
      WO2016089920-A1   BROAD INST INC (BROD);  BAYLOR COLLEGE MEDICINE (BAYU)   LIEBERMAN-AIDEN E, RAO S, STAMENOVA E, DUDCHENKO O, LANDER E
      WO2017219027-A1   BROAD INST INC (BROD);  MASSACHUSETTS INST TECHNOLOGY (MASI);  HARVARD COLLEGE (HARD);  ZHANG F (ZHAN-Individual);  ABUDAYYEH O O (ABUD-Individual);  GOOTENBERG J (GOOT-Individual);  LANDER E S (LAND-Individual)   ZHANG F, ABUDAYYEH O O, GOOTENBERG J, LANDER E S
   US2018340218-A1
      US20170032119-A1      
      US20170362644-A1      
   US2018340219-A1
      US20170032119-A1      
      US20170362644-A1      
   US10266887-B2
      US20170321198-A1      
      US20110111517-A1      
      US20120238008-A1      
      US20130190196-A1      
      US20140356867-A1      
      US20150065821-A1      
      US20150342509-A1      
      US20160282354-A1      
      US20170362644-A1      
      US20180208976-A1      
      US20180208977-A1      
      US7057026-B2   BARNES C (BARN-Individual);  BALASUBRAMANIAN S (BALA-Individual);  LIU X (LIUX-Individual);  SWERDLOW H (SWER-Individual)   BARNES C, BALASUBRAMANIAN S, LIU X, SWERDLOW H
      US7405281-B2   PACIFIC BIOSCIENCES CALIFORNIA INC (PABI)   XU Y, WEGENER J, BIBILLO A
      WO2011109762-A1   UNIV SINGAPORE NAT (UNUS);  MASSACHUSETTS INST TECHNOLOGY (MASI)   LIM C T, HAN J, HOU H W, BHAGAT A A, VAN VLIET K J, LEE W C
      WO2014047561-A1   BROAD INST INC (BROD);  STEELMAN S (STEE-Individual);  NICOL R (NICO-Individual)   STEELMAN S, NICOL R
      WO2015148431-A1   IMMCO DIAGNOSTICS INC (IMMC-Non-standard)   SURESH L, MALYAVANTHAM K
      WO2016089920-A1   BROAD INST INC (BROD);  BAYLOR COLLEGE MEDICINE (BAYU)   LIEBERMAN-AIDEN E, RAO S, STAMENOVA E, DUDCHENKO O, LANDER E
      WO2017040316-A1   BROAD INST INC (BROD);  MASSACHUSETTS INST TECHNOLOGY (MASI);  SABETI P (SABE-Individual)   SABETI P, METSKY H, GNIRKE A, MATRANGA C, PARK D
      WO2017070605-A1   BROAD INST INC (BROD);  MASSACHUSETTS INST TECHNOLOGY (MASI);  ZHANG F (ZHAN-Individual);  SMARGON A (SMAR-Individual);  PYZOCHA N (PYZO-Individual);  COX D B T (COXD-Individual);  LANDER E S (LAND-Individual)   ZHANG F, SMARGON A, PYZOCHA N, COX D B T, LANDER E S
      WO2017075292-A1   BROAD INST INC (BROD);  MASSACHUSETTS INST TECHNOLOGY (MASI)   REGEV A, LANDER E S, CLEARY B, CONG L
      WO2017079699-A1   BROAD INST INC (BROD);  GEN HOSPITAL CORP (GEHO);  HUNG D (HUNG-Individual);  JOHNSON E (JOHN-Individual);  POULSEN B (POUL-Individual)   HUNG D, JOHNSON E, POULSEN B
      WO2017219027-A1   BROAD INST INC (BROD);  MASSACHUSETTS INST TECHNOLOGY (MASI);  HARVARD COLLEGE (HARD);  ZHANG F (ZHAN-Individual);  ABUDAYYEH O O (ABUD-Individual);  GOOTENBERG J (GOOT-Individual);  LANDER E S (LAND-Individual)   ZHANG F, ABUDAYYEH O O, GOOTENBERG J, LANDER E S
      WO2018035250-A1   BROAD INST INC (BROD);  MASSACHUSETTS INST TECHNOLOGY (MASI);  UNIV RUTGERS STATE NEW JERSEY (RUTF);  US DEPT HEALTH & HUMAN SERVICES (USSH)   COX D, KOONIN E V, MAKAROVA K S, PYZOCHA N, SCOTT D, SEVERINOV K, SHMAKOV S, SMARGON A, WOLF Y I, YAN W, ZHANG F
      WO2018107129-A1   BROAD INST INC (BROD);  MASSACHUSETTS INST TECHNOLOGY (MASI);  HARVARD COLLEGE (HARD)   ABUDAYYEH O, COLLINS J J, GOOTENBERG J, ZHANG F, LANDER E S
   US10266886-B2
      US20170321198-A1      
      US20110111517-A1      
      US20120238008-A1      
      US20130190196-A1      
      US20140356867-A1      
      US20150065821-A1      
      US20150342509-A1      
      US20160282354-A1      
      US20170362644-A1      
      US20180208976-A1      
      US20180208977-A1      
      US7057026-B2   BARNES C (BARN-Individual);  BALASUBRAMANIAN S (BALA-Individual);  LIU X (LIUX-Individual);  SWERDLOW H (SWER-Individual)   BARNES C, BALASUBRAMANIAN S, LIU X, SWERDLOW H
      US7405281-B2   PACIFIC BIOSCIENCES CALIFORNIA INC (PABI)   XU Y, WEGENER J, BIBILLO A
      WO2011109762-A1   UNIV SINGAPORE NAT (UNUS);  MASSACHUSETTS INST TECHNOLOGY (MASI)   LIM C T, HAN J, HOU H W, BHAGAT A A, VAN VLIET K J, LEE W C
      WO2014047561-A1   BROAD INST INC (BROD);  STEELMAN S (STEE-Individual);  NICOL R (NICO-Individual)   STEELMAN S, NICOL R
      WO2015148431-A1   IMMCO DIAGNOSTICS INC (IMMC-Non-standard)   SURESH L, MALYAVANTHAM K
      WO2016089920-A1   BROAD INST INC (BROD);  BAYLOR COLLEGE MEDICINE (BAYU)   LIEBERMAN-AIDEN E, RAO S, STAMENOVA E, DUDCHENKO O, LANDER E
      WO2017040316-A1   BROAD INST INC (BROD);  MASSACHUSETTS INST TECHNOLOGY (MASI);  SABETI P (SABE-Individual)   SABETI P, METSKY H, GNIRKE A, MATRANGA C, PARK D
      WO2017070605-A1   BROAD INST INC (BROD);  MASSACHUSETTS INST TECHNOLOGY (MASI);  ZHANG F (ZHAN-Individual);  SMARGON A (SMAR-Individual);  PYZOCHA N (PYZO-Individual);  COX D B T (COXD-Individual);  LANDER E S (LAND-Individual)   ZHANG F, SMARGON A, PYZOCHA N, COX D B T, LANDER E S
      WO2017075292-A1   BROAD INST INC (BROD);  MASSACHUSETTS INST TECHNOLOGY (MASI)   REGEV A, LANDER E S, CLEARY B, CONG L
      WO2017079699-A1   BROAD INST INC (BROD);  GEN HOSPITAL CORP (GEHO);  HUNG D (HUNG-Individual);  JOHNSON E (JOHN-Individual);  POULSEN B (POUL-Individual)   HUNG D, JOHNSON E, POULSEN B
      WO2017219027-A1   BROAD INST INC (BROD);  MASSACHUSETTS INST TECHNOLOGY (MASI);  HARVARD COLLEGE (HARD);  ZHANG F (ZHAN-Individual);  ABUDAYYEH O O (ABUD-Individual);  GOOTENBERG J (GOOT-Individual);  LANDER E S (LAND-Individual)   ZHANG F, ABUDAYYEH O O, GOOTENBERG J, LANDER E S
      WO2018035250-A1   BROAD INST INC (BROD);  MASSACHUSETTS INST TECHNOLOGY (MASI);  UNIV RUTGERS STATE NEW JERSEY (RUTF);  US DEPT HEALTH & HUMAN SERVICES (USSH)   COX D, KOONIN E V, MAKAROVA K S, PYZOCHA N, SCOTT D, SEVERINOV K, SHMAKOV S, SMARGON A, WOLF Y I, YAN W, ZHANG F
      WO2018107129-A1   BROAD INST INC (BROD);  MASSACHUSETTS INST TECHNOLOGY (MASI);  HARVARD COLLEGE (HARD)   ABUDAYYEH O, COLLINS J J, GOOTENBERG J, ZHANG F, LANDER E S
CR WO2018107129-A1
      ABUDEYYEH, 00 ET AL.: "C2c2 is a single-component programmable RNA-guided RNA-targeting CRISPR effector", SCIENCE, vol. 353, no. 6299, 2 June 2016 (2016-06-02), pages 1 - 23, XP055407082,relevantClaims[1, 3/1, 4],relevantPassages[; page 2, 4th paragraph; page 3, 2nd paragraph; page 8, 1st paragraph; DOI: 10.1126/science.aaf5573]
      GOOTENBERG, JS ET AL.: "Nucleic acid detection with CRISPR-Cas13A/C2c2", SCIENCE, vol. 356, no. 6336, 28 April 2017 (2017-04-28), pages 438 - 442, XP055481345, Retrieved from the Internet &lt;URL:doi: 10.1126/science.aam9321&gt;,relevantClaims[1-2, 3/1-2, 4]
   US10266887-B2
      Yen et al., Multicolored silver nanoparticles for multiplexed disease diagnositics: distinguishing dengue, yellow fever, and Ebola viruses 15 Lab on a Chip 1638-1641 (2015).
      International Search Report and Written Opinion for PCT International Application No. PCT/US17/65477, dated Feb. 7, 2018, 15.
      Abudayyeh, et al., C2c2 is a Single-Component Programmable RNA-Guided RNA-Targeting CRISPR Effector, Science, vol. 353, No. 6299, Aug. 5, 2016, 1-23.
      Gootenberg, et al., Nucleic Acid Detection with CRISPR-Cas13A/C2c2, Science, vol. 356, No. 6336, Apr. 28, 2017, 438-442.
      Office Action for U.S. Appl. No. 15/917, 546, dated Sep. 21, 2018, 15.
      Abudayyeh, et al., C2c2 is a Single-Component Programmable RNA-Guided RNA-Targeting CRISPR Effector, Science, vol. 353, No. 6299, Jun. 2, 2016, 557, aaf5573 1-9 Supplement.
      Allerson, et al., Fully 2-modified oligonucleotide duplexes with improved in vitro potency and stability compared to unmodified small interfering RNA, J. Med. Chem, Feb. 24, 2005: 48(4): 901-4, Feb. 24, 2005, 901-904.
      Das, et al., Ultra-portable, wirelss smartphone spectrometer for reapid, non-destructive testing of fruit ripeness, Nature.com/Scientific Reports, Sep. 8, 2016, 1-8.
      Deng, et al., CASFISH: CRISPR/Cas9-mediated in situ labeling of genomic loci in fixed cells, PNAS; 112(38), Sep. 22, 2015, 11870-11875.
      Dirks, et al., Triggered amplification by hyrbridization chain reaction, PNAS; 101(43), Oct. 26, 2004, 15275-15278.
      East-Seletsky, et al., Two Distinct RNase Activities of CRISPR-C2c2 Enable Guide-RNA Processing and RNA Detection, Nature, vol. 538, Oct. 13, 2016, 270-273.
      Hale, et al., RNA-Guided RNA Cleavage by a CRISPR RNA-Cas Protein, Cell; 139(5), Nov. 25, 2009, 945-956.
      Hale, et al., Target RNA capture and cleavage by the Cmr type III-B CRISP-Cas effector complex, Genes & Developmentl; 28, Sep. 29, 2014, 2532-2443.
      Heider, et al., DNA watermarks: A proof of concept, BMC Mol Biol; 9, 2008, 40-50.
      Hendel, et al., Chemically modified guide RNAs enhance CRISPR-Cas genome editing in human primary cells, Nat. Biotechnol; 33(9), Sep. 2015, 985-989.
      Jabado, et al., Comprehensive viral oligonucleotide proble design using conserved protein regions, Nucleic Acids Research; 36(1), Jan. 2008, e3: 1-13.
      Jabado, et al., Greene SCPrimer: a rapid comprehensive tool for designing degenerat primers from multiple sequence alignments, Nucleic Acids Research; 34(22), Nov. 28, 2006, 6605-6611.
      Kelley, et al., Versatility of chemically synthesized guide RNAs for CRISPR-Cas9 genome editing, Journal of Biotechnology; 233, Sep. 10, 2016, 74-83.
      Lee, et al. Synthetically modified guide RNA and donor DNA are a versatile platform for CRISPR-Cas9 engineering, eLIFE 2017; 6: e25312, May 2, 2017, e2532: 1-23.
      Li, et al. Dramatic enhancement of the detection limits of bioassays via ultrafast deposition of polydopamine, Nat Biomed Eng. 2017; 1, 2017, 1-28.
      Lu, et al., Ultra-sensitive Calorimetric Assay System Based on the Hybridization Chain Reaction-Triggered Enzyme Cascade Amplification, ACS Appl Mater Interfaces ; 9(1), Jan. 1, 2017, 167-175.
      Nagrath, et al., Isolation of rare circulating tumour cells in cancer patients by microchip technology, Nature; 450 (7173), Dec. 20, 2007, 1235-1239.
      Pardee, et al., Paper-based Synthetic Gene Networks, Cell; 159 (4), Nov. 6, 2014, 940-954.
      Pardee, et al., Rapid, Low-Cost Detection of Zika Virus Using Programmable Biomolecular Components, Cell; 165, May 19, 2016, 1255-1266.
      Phillippy, et al., Efficient oligonucleotide probe selection for pan-genomic tiling arrays, BMC Bioinformatics 2009; 10: 293, Sep. 16, 2009.
      Platt, et al., CRISPR-Cas9 Knockin Mice for Genome Editing and Cancer Modelling, Cell; 159(2), Oct. 9, 2014, 440-455.
      Samai, et al., Co-transcriptional DNA and RNA cleavage during type III CRISPR-Cas immunity, Cell; 161(5), May 21, 2015, 1164-1174.
      Shafiee, et al., Paper and Flexible Substrates as Materials for Biosensing Platforms to Detect Multiple Biotargets, Scientific Reports; 5, Mar. 6, 2015, 1-9.
      Shmakov, et al., Discovery and functional characterization of diverse Class 2 CRISPR-Cas System, Mol Cell; 60(3), Nov. 5, 2015, 385-397.
      Shmakov, et al., Diversity and evolution of class 2 CRISPR-Cas Systems, Nat Rev Microbiol: 15(3), Mar. 2017, 169-182.
      Shoffner, et al., CHIP PCr. I. Surface Passivation of Microfabricated Silicon-Glass Chips for PCR, Nucleic Acids Research; 24(2), Jan. 1, 1996, 375-379.
      Smargon, et al., Cas 13B is a Type VI-B CRISPR-associated RNA-Guided RNAse differentially regulated by accessory proteins Csx27 and Csx28, Mol Cell; 65(4), Feb. 16, 2017, 618-630.
      Song, et al., Non-covalent Fluorescent Labeling of Hairpin DNA Probe Coupled wiht Hybridization Chain Reaction for Sensitive DNA Detection, Appl Spectrosc; 70(4), Apr. 2016, 688-694.
      Sorek, et al.' CRISPR-Mediated Adaptive Immune Systems in Bacteria and Archaea, Annual Review of Biochemistry. vol. 82, 2013, 237-266.
      St John, et al., Existing and Emerging Technologies for Point-of-Care Testing, Clin Biochem Rev; 35(3), 2014, 155-167.
      Vashist, et al., Commercial Smartphone-Based Devices and Smart Applications for Personalized Healthcare Monitoring and Management, Diagnostics, 2014, pp. 104-128.
      Wang, et al., An enzyme-free colorimetric assay using hybridization chain reaction amplification and split aptamers, Analyst, Nov. 2015, pp. 7657-7662.
      Wang, et al., Flexible substrate-based devices for point-of-care diagnostics, Trends Biotechnol., Author manuscript, Nov. 2016, pp. 909-921.
      Wang, et al., Structure-Switching Aptamer Triggering Hybridization Chain Reaction on the Cell Surface for Activatable Theranostics, Analytical Chemistry, 2015, pp. 6470-6474.
      Zetsche, et al., Cpf1 is a Single RNA-Guided Endonuclease of a Class 2 CRISPR-Cas System, Cell, vol. 163, Sep. 25, 2015, 759-771.
      Zhao, et al. Signal amplification of glucosamine-6-phosphate based on ribozyme glmS, Biosens. Bioelectron., Dec. 2014, pp. 337-342.
   US10266886-B2
      East-Seletsky et al., Two distinct RNase activities of CRISPR-C2c2 enable guideRNA processing and RNA detection 538 Nature 270-273 and Methods (Oct. 12, 2016).
      Sorek et al., CRISPR-Mediated Adaptive Immune Systems in Bacteria and Archaea 82 Annual Review of Biochemistry 237-266 (2013).
      Abudayyeh et al., C2c2 is a single-component programmable RNA-guided RNA-targeting CRISPR effector 353(6299) Science 557, aaf5573 1-9 and Supplement (Jun. 2, 12016).
      Zetsche et al., Cpf1 Is a Single RNA-Guided Endonuclease of a Class 2 CRISPR-Cas System 163 Cell 759-771 (Sep. 25, 2015).
      International Search Report and Written Opinion for PCT International Application No. PCT/US17/65477, dated Feb. 7, 2018, 15.
      Abudayyeh, et al., C2c2 is a Single-Component Programmable RNA-Guided RNA-Targeting CRISPR Effector, Science, vol. 353, No. 6299, Aug. 5, 2016, 1-23.
      Gootenberg, et al., Nucleic Acid Detection with CRISPR-Cas13A/C2c2, Science, vol. 356, No. 6336, Apr. 28, 2017, 438-442.
      Allerson, et al., Fully 2-modified oligonucleotide duplexes with improved in vitro potency and stability compared to unmodified small interfering RNA, J. Med. Chem, Feb. 24, 2005: 48(4): 901-4, Feb. 24, 2005, 901-904.
      Das, et al., Ultra-portable, wireless smartphone spectrometer for rapid, non-destructive testing of fruit ripeness, Nature.com/Scientific Reports, Sep. 8, 2016, 1-8.
      Deng, et al., CASFISH: CRISPR/Cas9-mediated in situ labeling of genomic loci in fixed cells, PNAS; 112(38), Sep. 22, 2015, 11870-11875.
      Dirks, et al., Triggered amplification by hyrbridization chain reaction, PNAS; 101(43), Oct. 26, 2004, 15275-15278.
      Hale, et al., RNA-Guided RNA Cleavage by a CRISPR RNA-Cas Protein, Cell; 139(5), Nov. 25, 2009, 945-956.
      Hale, et al., Target RNA capture and cleavage by the Cmr type III-B CRISP-Cas effector complex, Genes & Developmentl; 28, Sep. 29, 2014, 2532-2443.
      Heider, et al., DNA watermarks: A proof of concept, BMC Mol Biol; 9, 2008, 40-50.
      Hendel, et al., Chemically modified guide RNAs enhance CRISPR-Cas genome editing in human primary cells, Nat. Biotechnol; 33(9), Sep. 2015, 985-989.
      Jabado, et al., Comprehensive viral oligonucleotide proble design using conserved protein regions, Nucleic Acids Research; 36(1), Jan. 2008, e3: 1-13.
      Jabado, et al., Greene SCPrimer: a rapid comprehensive tool for designing degenerat primers from multiple sequence alignments, Nucleic Acids Research; 34(22), Nov. 28, 2006, 6605-6611.
      Kelley, et al., Versatility of chemically synthesized guide RNAs for CRISPR-Cas9 genome editing, Journal of Biotechnology; 233, Sep. 10, 2016, 74-83.
      Lee, et al., Synthetically modified guide RNA and donor DNA are a versatile platform for CRISPR-Cas9 engineering, eLIFE 2017; 6: e25312, May 2, 2017, e2532: 1-23.
      Li, et al. Dramatic enhancement of the detection limits of bioassays via ultrafast deposition of polydopamine, Nat Biomed Eng. 2017; 1, 2017, 1-28.
      Lu, et al., Ultra-sensitive Colorimetric Assay System Based on the Hybridization Chain Reaction-Triggered Enzyme Cascade Amplification, ACS Appl Mater Interfaces ; 9(1), Jan. 1, 2017, 167-175.
      Nagrath, et al., Isolation of rare circulating tumour cells in cancer patients by microchip technology, Nature; 450 (7173), Dec. 20, 2007, 1235-1239.
      Pardee, et al., Paper-based Synthetic Gene Networks, Cell; 159 (4), Nov. 6, 2014, 940-954.
      Pardee, et al., Rapid, Low-Cost Detection of Zika Virus Using Programmable Biomolecular Components, Cell; 165, May 19, 2016, 1255-1266.
      Phillippy, et al., Efficient oligonucleotide probe selection for pan-genomic tiling arrays, BMC Bioinformatics 2009; 10: 293, Sep. 16, 2009.
      Platt, et al., CRISPR-Cas9 Knockin Mice for Genome Editing and Cancer Modelling, CELL; 159(2), Oct. 9, 2014, 440-455.
      Samai, et al., Co-transcriptional DNA and RNA cleavage during type III CRISPR-Cas immunity, Cell; 161(5), May 21, 2015, 1164-1174.
      Shafiee, et al., Paper and Flexible Substrates as Materials for Biosensing Platforms to Detect Multiple Biotargets, Scientific Reports; 5, Mar. 6, 2015, 1-9.
      Shmakov, et al., Discovery and functional characterization of diverse Class 2 CRISPR-Cas System, Mol Cell; 60 (30, Nov. 5, 2015, 385-397.
      Shmakov, et al., Diversity and evolution of class 2 CRISPR-Cas Systems, Nat Rev Microbiol: 15(3), Mar. 2017, 169-182.
      Shoffner, et al., CHIP PCr. I. Surface Passivation of Microfabricated Silicon-Glass Chips for PCR, Nucleic Acids Research; 24(2), Jan. 1, 1996, 375-379.
      Smargon, et al., Cas 13B is a Type VI-B CRISPR-associated RNA-Guided RNAse differentially regulated by accessory proteins Csx27 and Csx28, Mol Cell; 65(4), Feb. 16, 2017, 618-630.
      Song, et al., Non-covalent Fluorescent Labeling of Hairpin DNA Probe Coupled wiht Hybridization Chain Reaction for Sensitive DNA Detection, Appl Spectrosc; 70(4), Apr. 2016, 688-694.
      St John, et al., Existing and Emerging Technologies for Point-of-Care Testing, Clin Biochem Rev; 35(3), 2014, 155-167.
      Vashist, et al., Commercial Smartphone-Based Devices and Smart Applications for Personalized Healthcare Monitoring and Management, Diagnostics, 2014, pp. 104-128.
      Wang, et al., An enzyme-free colorimetric assay using hybridization chain reaction amplification and split aptamers, Analyst, Nov. 2015, pp. 7657-7662.
      Wang, et al., Flexible substrate-based devices for point-of-care diagnostics, Trends Biotechnol., Author manuscript, Nov. 2016, pp. 909-921.
      Wang, et al., Structure-Switching Aptamer Triggering Hybridization Chain Reaction on the Cell Surface for Activatable Theranostics, Analytical Chemistry, 2015, pp. 6470-6474.
      Zhao, et al., Signal amplification of glucosamine-6-phosphate based on ribozyme glmS, Biosens. Bioelectron., Dec. 2014, pp. 337-342.
DN 93605-0-0-0-A K; 105730-0-0-0-A D K; 184611-0-0-0-A K; 184616-0-0-0-D K M; 184610-0-0-0-D K M; 96357-0-0-0-D K M; 131322-0-0-0-D K M; 556-0-1-0-D K M
CI RA00NS-A K; RA012P-A D K; RA00H1-A K; RA00H3-D K M; RA013I-D K M; RA05XY-D K M; R10200-D K M; R00226-D K M
RG 0226-S
UT DIIDW:201846551F
ER"""


import re

print( re.split(";|,",re.search(r"PN(.+)?\nTI",text).group(1))[0].strip())